legislation-parameters.md

```diff
@@ -0,0 +1,109 @@
+# Legislation parameters
+
+## Definition
+
+The legislation parameters are stored in XML file in the directory `parameters` (see the [directory `parameters`  of OpenFisca-France](https://github.com/openfisca/openfisca-france/blob/master/openfisca_france/parameters)).
```

Comment:
- “in XML fileS”
- Double space in “`parameters`  of”.
- “see the directory `parameters`  of OpenFisca-France” -> “example in OpenFisca-France”